src/lbaf/IO/lbsVTDataReader.py

```diff
@@ -164,7 +164,7 @@ def read_iteration(self, phase_id: int) -> list:
         # Return populated list of ranks
         return rank_list
 
-    def json_reader(self, returned_dict: dict, phase_ids, node_id: int) -> tuple:
+    def json_reader(self, returned_dict: dict, phase_id: int, node_id: int) -> tuple:
```

Comment:
@ppebay probably this change here is causing Unit Tests error: `TypeError: json_reader() got an unexpected keyword argument 'phase_ids'`
![Zrzut ekranu z 2022-11-16 11-00-32](https://user-images.githubusercontent.com/47597569/202163859-8c06803d-b174-47e8-b014-2dad3983b3e5.png)

Reply:
I assume that was resolved by your latest commit thanks